.buildkite/scripts/pipelines/security_solution_quality_gate/upload_runtime_info.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+echo "$KIBANA_DOCKER_PASSWORD" | docker login -u "$KIBANA_DOCKER_USERNAME" --password-stdin docker.elastic.co
+
+if [ "$KIBANA_LATEST" = "1" ]; then
```

Comment:
this is a bit confusing, when KIBANA_LATEST=1 then we're _not_ pulling the latest?

Reply:
Hmmm.... I may need to change the variable name. It is indeed confusing.

Reply:
I renamed KIBANA_LATEST to KIBANA_CURRENT to reflect the fact that I will be building a new image with the current commit every time the flag exists.